tests/testsuite/vendor.rs

```diff
@@ -956,10 +1521,6 @@ fn git_deterministic() {
 # will likely look very different (and much more reasonable).
 # See Cargo.toml.orig for the original contents.
 
-bin = []
-test = []
-bench = []
```

Comment:
I am sorry that I didn't even notice this when reviewing that PR.

Reply:
Notice which?